graphqlCodegen.ts

```diff
@@ -1,7 +1,16 @@
 import { CodegenConfig } from "@graphql-codegen/cli";
 
+const subgraphApiKey = process.env.VITE_SUBGRAPH_API_KEY;
```

Comment:
this should be `import.meta.env.VITE_SUBGRAPH_API_KEY` https://vitejs.dev/guide/env-and-mode.html